README.md

```diff
@@ -1,6 +1,7 @@
+![logo jpg](https://user-images.githubusercontent.com/71459989/135807893-2dda4ce6-f9b8-4232-a893-631aef3f2183.jpg)
```

Comment:
Our Readme is already updated by someone else. But I like that logo you add can you add this to our new readme file.